.github/workflows/cypress.yaml

```
@@ -8,17 +8,45 @@ on:
+
+      - name: Save build folder
+        uses: actions/upload-artifact@v4
+        with:
+          name: build
+          if-no-files-found: error
+          path: build
+
+  parallel-job:
+    if: github.repository == 'ohcnetwork/care_fe'
+    permissions:
+      write-all: true
```

actionlint (1.7.4)

unknown permission scope "write-all". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)